src/utils/redux.js

```diff
@@ -33,4 +33,4 @@ export const isStateGoingBack = (cur: GlobalState, prev: GlobalState): boolean =
   || isNavTransitionGoingOn(cur, prev)
   || isEqual(cur, prev);
 
-export const connectPreserveOnBackOption = { areStatesEqual: isStateGoingBack };
+export const connectPreserveOnBackOption = () => ({ areStatesEqual: isStateGoingBack });
```

Comment:
This is a significant regression that might cause performance issues (may or may not, good to investigate).

The first version creates a constant and it is reused. The new version produces a new object on each call.

Reply:
> The first version creates a constant and it is reused. The new version produces a new object on each call.

To overcome this we can make another version of `connectPreserveOnBackOption` which only depends on `isStateGoingBack` and not on `isNavTransitionGoingOn`